src/otx/config/data/openvino.yaml

```diff
@@ -0,0 +1,18 @@
+defaults:
+  - default
+
+train_subset:
+  transform_lib_type: TORCHVISION
+  transforms:
+    - _target_: otx.core.data.transform_libs.torchvision.Identity
+  batch_size: 1
+val_subset:
+  transform_lib_type: TORCHVISION
+  transforms:
+    - _target_: otx.core.data.transform_libs.torchvision.Identity
```

Comment:
Why do we have to use underscores here? It looks as some kind of irregularity

Reply:
It comes from Hydra syntaxis. With __target__ we initialize target class
https://hydra.cc/docs/advanced/instantiate_objects/overview/#instantiation-of-builtins